src/controller/RouteController.tsx

```diff
@@ -0,0 +1,18 @@
+
+import HeadMenu from "../components/headMenu";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { BooksMain } from "../routes/books";
+import { CartMain } from "../routes/cart";
+
+function RouteController(): React.ReactElement {
+
+  return <Router>
+    <HeadMenu />
```

Comment:
This component is reserved only for routes. Any other element must be specified in the App.tsx file.

Reply:
But Sir Header is something that is common should we not keep it at one place that's why it is done

Reply:
We place the common code in App.tsx file.

Reply:
Sure Sir Adding those changes